profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/tmgmt_poetry_mock.behat.inc

```diff
@@ -99,7 +99,7 @@ class TMGMTPoetryMockSubContext extends RawDrupalContext implements DrupalSubCon
   /**
    * Remove pending translations in a scenario tagged with @poetry.
    *
-   * @AfterScenario @poetry
+   * @poetry
```

Comment:
Its normal because the Afterscenario is not needed anymore as. The module is unsinstalled now after the scenario and the uninstall hook contains the process defined here.